index.html

```diff
@@ -146,6 +146,26 @@ <h1> <img src="images/Rectangle 57.png" alt=""> Skills</h1>
             </ul>
         </div>
     </section>
+    <section class="contact-form">
+        <h2>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/xjvdkobw" method="POST">
+            <input type="text" name="name" placeholder="Full name" style="margin-bottom: 33px;" maxlength="30" required>
+            <input type="email" name="email" placeholder="Email address" style="margin-top: 0px;" required>
+            <textarea name="msg" cols="30" rows="5" maxlength="500" required placeholder="Enter your text here"></textarea>
+            <button type="submit">Get in touch</button>
+        </form>
+        <hr>
+        <ul class="social-media">
+
+            <li><a href="#"><img src="images/github.png" alt=""></a></li>
+            <li><a href="#"><img src="images/linkedin.png" alt=""></a></li>
+            <li><a href="#"><img src="images/Vector.png" style="margin-top: 12px;" alt=""></a></li>
+            <li><a href="#"><img src="images/twitter.png" alt=""></a></li>
+            <li><a href="#"><img src="images/medium.png" alt=""></a></li>
+        </ul>
+    </section>
```

Comment:
- [Optional] You did great making this design 👏  However, I strongly suggest you adjust it to suite the color pixel given as provided on the [FIGMA](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=SCAYEfPbWTD12COc-0) Design. Good work 👏 

# Your design:
![image](https://user-images.githubusercontent.com/50990925/213086992-b1d01f02-a08a-4c24-b107-ed278bebb07e.png)

# Figma Design
![image](https://user-images.githubusercontent.com/50990925/213087038-f7f92c63-f67b-4643-969e-6d092a4d89b9.png)